hrms/hr/doctype/full_and_final_statement/full_and_final_statement.json

```diff
@@ -221,11 +221,12 @@
    "report": 1,
    "role": "HR Manager",
    "share": 1,
+   "submit:" 1, 
```

Comment:
You will also have to update the `modified` timestamp key in the file for the changes to sync on site migration